_theme_dev/src/js/theme/components/useHoverIntent.js

```diff
@@ -0,0 +1,235 @@
+/**
+ * Creates a hover intent tracker for a given element.
```

Comment:
Ha, I remember using jQuery hover intent years ago. Did you decide to create your own version or is that base on something?

Reply:
@kpodemski yea, that's actually rewritten jQuery hoverIntent to vanilla.js but still I am not sure it's final implementation. Whole front end I made only for POC to find the best way to load submenu.